oxd-server/src/main/java/org/gluu/oxd/server/op/GetAuthorizationUrlOperation.java

```diff
@@ -48,6 +49,12 @@ public IOpResponse execute(GetAuthorizationUrlParams params) throws Exception {
             scope.addAll(site.getScope());
         }
 
+        if (Utils.hasAuthorizationRedirectUri(site.getRedirectUris(), params.getAuthorizationRedirectUri())) {
```

Comment:
drop this snippet completely. /get-authorization-url must not modify registration.

Reply:
Changes done.